webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -7,6 +7,7 @@
 import json
 import random
 import urlparse
+from builtins import int
```

Comment:
`int` will always be in the namespace--you don't need to import it.

```
$ python
Python 2.7.15rc1 (default, Apr 15 2018, 21:51:34) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> int
<type 'int'>
>>> 
```

```
$ python3
Python 3.6.5 (default, Apr  1 2018, 05:46:30) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> int
<class 'int'>
>>> 
```

As a side note, Python has a read-eval-print-loop (often called "repl") that lets you play with the language and code iteratively. I often use this to test things out. You can usually run it by typing "python" in a terminal/shell. Some systems have Python 2 and Python 3 installed and "python" will be Python 2  and "python3" will be Python 3. Python will tell you which version it is at startup.

I don't know if that's helpful to know or not.